lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.scss

```diff
@@ -1,50 +1,35 @@
 adf-search-chip-autocomplete-input {
     .adf-chip-list {
         width: 100%;
-    }
-
-    .mat-form-field-appearance-outline .mat-form-field-outline {
-        top: 0;
-    }
 
-    .mat-form-field-infix {
-        border: none;
+        /* stylelint-disable-next-line */
+        .mat-form-field-infix {
```

Comment:
wasn't possible to remove this one?

Reply:
No, I've tried many approaches, but unfortunately div with this class is not approachable any other way. And it's necessary to change it, as there are many styling changes needed in that component. Do you have any idea how to handle such a situation?